manifests/crc/default-db-included-installation/aks/kruize-crc-aks.yaml

```diff
@@ -156,7 +156,7 @@ spec:
     spec:
       containers:
         - name: kruize
-          image: quay.io/kruize/autotune_operator:0.5
+          image: quay.io/kruize/autotune_operator:0.6
```

Comment:
Need to update the image tag in the `kruizecronjob` and `kruizedeletejob` section as well